### DISCUSSION TITLE
Not able to build docker-compose build and docker-compose up of celery

### DISCUSSION THREAD

**Discussion by Sauraj1234** [celery/celery#8086] [General]
Hi ,

Not able to build docker image for celery using 
Dockerfile inside docker folder and i am in the master 
branch 
Here are some screenshots 

<img width="1791" alt="Screenshot 2023-02-25 at 3 04 50 PM" src="https://user-images.githubusercontent.com/31471612/221350047-1a160e0d-6581-4c85-a300-c27d89290133.png">

<img width="1791" alt="Screenshot 2023-02-25 at 3 05 21 PM" src="https://user-images.githubusercontent.com/31471612/221350067-4cd439a1-e158-4f1d-ba4e-783ab5b526ff.png">

**Comment by Nusnus** [comment]
Fixed in https://github.com/celery/celery/pull/8275
